nc_test4/tst_bug1442.c

```diff
@@ -0,0 +1,73 @@
+/*! \file
+
+Copyright 2019
+University Corporation for Atmospheric Research/Unidata.
+
+See \ref copyright file for more info.
+
+*/
+#include <nc_tests.h>
+#include "err_macros.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <netcdf.h>
+
+#define FILENAME "tst_bug1442nc"
```

Comment:
I believe it must be tst_*.nc in order to be automatically cleaned. You are missing the "." before the nc.

Reply:
done